src/test/groovy/graphql/language/AstMultiZipperTest.groovy

```diff
@@ -0,0 +1,98 @@
+package graphql.language
+
+import graphql.TestUtil
+import spock.lang.Specification
+
+
+class AstMultiZipperTest extends Specification {
+
+
+    def "toRootNode with one zipper"() {
+        def rootNode = TestUtil.parseQuery("{root{midA{leafA leafB} midB{leafC leafD}}}")
+        def operationDefinition = rootNode.children[0] as OperationDefinition
+
+        def rootField = operationDefinition.selectionSet.children[0] as Field
+        def midA = rootField.selectionSet.children[0] as Field
+        def leafA = midA.selectionSet.children[0] as Field
+        def leafB = midA.selectionSet.children[0] as Field
+
```

Comment:
I think the indices here are wrong.

     def leafB = midA.selectionSet.children[1] as Field

here and below

Reply:
fixed